config.yml

```diff
@@ -88,7 +88,8 @@ pec:
       human: "syn20781820"
       animal: "syn21053116"
     biospecimen_templates:
-      general: "syn20768131"
+      mouse or other animal model: "syn20768131"
```

Comment:
This should be: ` animal model: syn20768131`

The PEC config has the species "human" and "animal model". These species dictate which species options appear in the app for the user and, ultimately, which species-based templates are available.

https://github.com/kelshmo/dccvalidator/blob/bcb4b11eb608439e4e9b1cceb43ae9a4be30aebd/config.yml#L97-L99

Sorry for the confusion. Please let me know if you have any questions.